client/blocks/comment-detail/index.jsx

```diff
@@ -68,8 +69,10 @@ export class CommentDetail extends Component {
 	}
 
 	deleteCommentPermanently = () => {
-		const { commentId, deleteCommentPermanently } = this.props;
-		deleteCommentPermanently( commentId );
+		const { commentId, deleteCommentPermanently, translate } = this.props;
+		if ( window.confirm( translate( 'Delete this comment permanently?' ) ) ) {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Delete this comment permanently' )` *ES Score: 18*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/comments/delete-confirmation)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).